docs/sources/installation/mac.md

```diff
@@ -314,6 +314,31 @@ section.
 	The installer places Boot2Docker in your "Applications" folder.
 
 
+## Uninstall Boot2Docker
+
+1. Go to the [boot2docker/osx-installer ](
+   https://github.com/boot2docker/osx-installer/releases/latest) release page. 
+
+2. Download the source code by clicking `Source code (zip)` or
+   `Source code (tar.gz)` in the "Downloads" section.
+
+3. Extract the source code.
+
+4. Open a terminal on your local machine.
+
+5. Change to the directory where you extracted the source code:
+
+		$ cd <path to extracted source code>
+
+6. Make sure the uninstall.sh script is executable:
+
+		$ chmod +x uninstall.sh
+
+7. Run the uninstall.sh script:
+
+		$ ./uninstall.sh
```

Comment:
Note; perhaps 8 spaces is needed because of the numbered list here (in that case, ignore my comment)